### ISSUE TITLE
Wrong test to apply useRelativePath

### ISSUE THREAD

**Issue by landru29** [webpack-contrib/file-loader#232]
https://github.com/webpack-contrib/file-loader/blob/master/src/index.js#L40

should be 
```
if (!/^\//.test(relativePath)) {
```

If in a CSS file you have:
```
div.jsoneditor-tree button {
  background: transparent url(img/jsoneditor-icons.svg);
}

.flag-icon-ad {
  background-image: url(../flags/4x3/ad.svg);
}

```

the following  conf will not work. `../flags/4x3/ad.svg` will be in relative but `img/jsoneditor-icons.svg` is ??
```
/* global __dirname */

const path = require("path");

const config = {
    entry: {
        app: path.resolve(__dirname, "client/app/app.module.js"),
    },
    output: {
        path: path.resolve(__dirname, ".tmp/app/bundle"),
        filename: "[name].js",
        publicPath: "app/"
    },
    module: {
        rules: [
            {
                test: /\.(jpe?g|png|gif|svg)/i,
                use: [
                    {
                        loader: "file-loader",
                        options: {
                            outputPath: "..",
                            name: "[name].[ext]",
                            useRelativePath: true
                        }
                    }
                ]
            },
            {
                test: /\.css$/,
                use: [
                    {
                        loader: "style-loader",
                        options: {
                            sourceMap: true
                        }
                    },
                    {
                        loader: "css-loader",
                        options: {
                            sourceMap: true
                        }
                    }
                ]
            }
        ]
    }
};
```

**Comment by michael-ciniawsky** [comment]
`directory/file.ext` is absolute ? => `'./directory/file.ext'`

**Comment by michael-ciniawsky** [comment]
`img/jsoneditor-icons.svg` is treated as an absolute path by the `fs`, doesn't the `css-loader` prefix it with `'./'`  ? If it doesn't it's a bug in `css-loader`

**Comment by landru29** [comment]
No, in a css file, img/foo.svg is treated as relative

**Comment by michael-ciniawsky** [comment]
https://github.com/webpack-contrib/css-loader/blob/master/lib/loader.js#L108
https://github.com/webpack/loader-utils/blob/master/lib/stringifyRequest.js#L33

```js
const path = require('path')

console.log(path.isAbsolute('img/foo.svg'))
// => false (I stand corrected here :))
```

So it will be treated as a relative path (`file-loader` will 'see' `'./img/foo.svg'`), I don't understand what isn't working here atm could you rephrase ?
